code/modules/mob/living/carbon/human/species/station/traits_vr/negative.dm

```diff
@@ -133,3 +133,16 @@
 /datum/trait/colorblind/para_taj/apply(var/datum/species/S,var/mob/living/carbon/human/H)
 	..(S,H)
 	H.add_modifier(/datum/modifier/trait/colorblind_taj)
+
+/datum/trait/coldprone
+	name = "Cold prone"
```

Comment:
Should be more in-line with the other traits, IE Cold-Blooded, meaning you are more uncomfortable in cold environments. Moreover, we already HAVE cold-blooded and I don't see quite how this trait is any different?

Reply:
Cold-blooded/Hot-blooded ONLY change comfort levels, while these actually affect how much cold the body can handle before taking damage.